internal/checker/checker.go

```diff
@@ -13576,7 +13576,27 @@ func (c *Checker) getTargetOfModuleDefault(moduleSymbol *ast.Symbol, node *ast.N
 }
 
 func (c *Checker) reportNonDefaultExport(moduleSymbol *ast.Symbol, node *ast.Node) {
-	// !!!
+	if moduleSymbol.Exports != nil && moduleSymbol.Exports[node.Symbol().Name] != nil {
```

Comment:
```suggestion
	if moduleSymbol.Exports[node.Symbol().Name] != nil {
```

A nil map is readable, so this is okay.